packages/lit-dev-server/package.json

```diff
@@ -8,9 +8,38 @@
   "type": "module",
   "main": "lib/server.js",
   "scripts": {
-    "build": "npm run build:ts",
-    "build:ts": "../../node_modules/.bin/tsc",
-    "start": "node lib/server.js"
+    "build": "wireit",
+    "build:ts": "wireit",
+    "start": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:ts"
+      ]
+    },
+    "build:ts": {
+      "dependencies": [
+        "../lit-dev-tools-cjs:build"
+      ],
+      "files": [
+        "src/**",
+        "package.json",
```

Comment:
Should this be `tsconfig.json` instead of `package.json`? Same below.

Reply:
+1, done